exercises/binary-search/build.gradle

```diff
@@ -0,0 +1,18 @@
+apply plugin: "java"
+apply plugin: "eclipse"
+apply plugin: "idea"
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  testCompile "junit:junit:4.12"
+  testCompile "org.assertj:assertj-core:3.2.0"
```

Comment:
You are right, @javaeeeee, we haven't completely removed "not technically necessary" third party dependencies.  Sorry for the confusion that caused.